tests/rest/client/v2_alpha/test_auth.py

```diff
@@ -499,3 +499,221 @@ def test_ui_auth_fails_for_incorrect_sso_user(self):
         self.delete_device(
             self.user_tok, self.device_id, 403, body={"auth": {"session": session_id}}
         )
+
+
+class RefreshAuthTests(unittest.HomeserverTestCase):
```

Comment:
I only did REST API tests there. This does not cover some cases like SSO and app services logins yet